tensorflow/contrib/tpu/profiler/pip_package/setup.py

```diff
@@ -52,12 +52,24 @@
         #   4 - Beta
         #   5 - Production/Stable
         'Development Status :: 3 - Alpha',
-
-        # Indicate who your project is intended for
+        
         'Intended Audience :: Developers',
-        'Topic :: Software Development',
+        'Intended Audience :: Education',
+        'Intended Audience :: Science/Research',
+        
         'License :: OSI Approved :: Apache Software License',
+        
+        'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
```

Comment:
In theory, we should list all them. These flags: Programming Language :: Python :: 3.X are useful if you want to request the PyPI API and be able to know when a library changes and which version of python is supported.
You tell me @gunan and I perform the changes if you think flagging Py3.4 and Py3.5 is something desirable. I would personally think it is better because I don't see any reason not to do it ;)